jaeger-core/src/main/java/com/uber/jaeger/reporters/RemoteReporter.java

```diff
@@ -93,13 +93,13 @@ public void close() {
       boolean added = commandQueue
           .offer(new CloseCommand(), closeEnqueueTimeout, TimeUnit.MILLISECONDS);
       if (added) {
-        queueProcessorThread.join();
+        queueProcessorThread.join(10000);
       } else {
         log.warn("Unable to cleanly close RemoteReporter, command queue is full - probably the"
             + " sender is stuck");
       }
     } catch (InterruptedException e) {
-      return;
+      // ignored
```

Comment:
Seeing that we might be running into races, I think we should log this exception.